evap/evaluation/models_logging.py

```diff
@@ -202,32 +202,40 @@ def log_instance_delete(self):
         changes = self._get_change_data(InstanceActionType.DELETE)
         self._update_log(changes, InstanceActionType.DELETE)
 
-    def _update_log(self, changes, action_type: InstanceActionType):
+    def _create_log_entry(self, action_type=InstanceActionType.CREATE):
```

Comment:
Since this doesn't actually write to `self`, maybe this should not be a method and instead a classmethod that also takes an instance ~_or_ a constructor on `LogEntry`~ (edit: need `thread`)?

Reply:
Didn't really change anything about the logic here.

Thinking about it, I think the current code is wrong, because the initialization of `thread` is done in the class declaration, which afaik will be done exactly once, shared for all threads. Then, all threads would share the same "local" context.

Reply:
Usage of `thread` is necessary and seems to be correct (contrary to what I first thought, see #1701 for details). Resolving this for now.